select/sdks/android/migration-guide.md

```diff
@@ -45,8 +45,7 @@ class MainActivity : AppCompatActivity() {
         Fidel.programId = "Your Program ID"
         
         /*
-         * It is recommended to store the SDK key on your backend and retrieve it
-         * before starting to use the SDK.
+         * We recommend storing the SDK key on your server. Please use one of your endpoints to retrieve it and then * configure the sdkKey property with it.
```

Comment:
There seems to be a small typo with an extra `*` there

Reply:
Thank you!